sdk/js/src/aptos/example.ts

```diff
@@ -0,0 +1,23 @@
+import { AptosAccount } from "aptos";
+import { AptosApi } from "./api";
+import { AptosClientWrapper } from "./client";
+import { AptosNetwork } from "./constants";
+import { hex } from "./utils";
+
```

Comment:
Maybe we could add a `testnet` example as well, so end users don't have to learn how to spin up a local validator before testing things out. 

Publishing a message through core bridge or doing a token transfer might be useful examples.